website/versioned_docs/version-7.0.0/plugin-proposal-do-expressions.md

```diff
@@ -9,7 +9,7 @@ original_id: babel-plugin-proposal-do-expressions
 
 > The `do { .. }` expression executes a block (with one or many statements in it), and the final statement completion value inside the block becomes the completion value of the do expression.
 
-from [You Don't Know JS](https://github.com/getify/You-Dont-Know-JS/blob/master/types%20%26%20grammar/ch5.md#statement-completion-values)
+from [You Don't Know JS](https://github.com/getify/You-Dont-Know-JS/blob/1st-ed/types%20%26%20grammar/ch5.md#statement-completion-values), Types & Grammar -> Chapter 5 -> Section Statement Completion Values
```

Comment:
I removed `website/versioned_docs/version-7.8.0/plugin-proposal-do-expressions.md` because it should had been backported.